src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -402,4 +406,31 @@ private function getServiceClass(ContainerBuilder $container, string $serviceId)
             return $definition->getClass();
         }
     }
+
+    private function registerRoutedMessages(ContainerBuilder $container): void
+    {
+        $mapping = $container->getDefinition('messenger.senders_locator')->getArgument(0) ?? [];
+        $receiverAliases = array_map(
+            static fn ($tagAttributes) => $tagAttributes[0]['alias'],
+            $container->findTaggedServiceIds('messenger.receiver')
+        );
+
+        foreach ($container->findTaggedServiceIds('messenger.routed_message') as $routedMessage) {
+            $routedMessage = $routedMessage[0];
```

Comment:
```suggestion
        foreach ($container->findTaggedServiceIds('messenger.routed_message') as [$routedMessage]) {
```

Reply:
I didn't know this syntax, definitely going to use this everyday 😄 Updated, thanks!